css/its-a-trap.css

```diff
@@ -11,17 +13,21 @@
     width: 430px;
 }
 
+* html #its-unwanted #main-feature,
 * html #its-a-trap #main-feature,
 * html #its-an-attack #main-feature { height: 265px; /* Hack for IE6 */ }
```

Comment:
Should there be IE6 fixes in here?

My understanding is that this page is invoked from inside the browser.